acceleration/TensorRT_inference_acceleration.ipynb

```diff
@@ -0,0 +1,930 @@
+{
```

Comment:
<p>my suggestion is to combine these three figure together, or unify the y-axis range (such as from 1.2 to 5.2) and put three figures into the same line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1215/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Review NB seems has some issue here. This comment is for figures shown in the `Benchmark the model inference time` section